src/server/controlchan/commands/cwd.rs

```diff
@@ -50,7 +50,7 @@ where
         let logger = args.logger;
 
         if let Err(err) = storage.cwd((*session.user).as_ref().unwrap(), path.clone()).await {
-            slog::warn!(logger, "CWD: Failed to cwd directory: {}", err);
+            slog::warn!(logger, "CWD: Failed to cwd directory {:?}: {} ", path, err);
```

Comment:
Suggest changing `CWD: Failed to cwd directory` to `CWD: Failed to change directory` as cwd already contains "directory"